model/client4.go

```diff
@@ -729,6 +729,17 @@ func (c *Client4) GetChannelMember(channelId, userId, etag string) (*ChannelMemb
 	}
 }
 
+// GetChannelMembersByIds gets all the channel members for a list of users on a channel.
```

Comment:
Can you change this to `GetChannelMembersByIds gets the channel members in a channel for a list of user Ids.`?